addons/account_asset/account_asset.py

```diff
@@ -357,124 +331,144 @@ def open_entries(self, cr, uid, ids, context=None):
         }
 
 
-class account_asset_depreciation_line(osv.osv):
+class account_asset_depreciation_line(models.Model):
     _name = 'account.asset.depreciation.line'
     _description = 'Asset depreciation line'
 
-    def _get_move_check(self, cr, uid, ids, name, args, context=None):
-        res = {}
-        for line in self.browse(cr, uid, ids, context=context):
-            res[line.id] = bool(line.move_id)
-        return res
-
-    _columns = {
-        'name': fields.char('Depreciation Name', required=True, select=1),
-        'sequence': fields.integer('Sequence', required=True),
-        'asset_id': fields.many2one('account.asset.asset', 'Asset', required=True, ondelete='cascade'),
-        'parent_state': fields.related('asset_id', 'state', type='char', string='State of Asset'),
-        'amount': fields.float('Current Depreciation', digits_compute=dp.get_precision('Account'), required=True),
-        'remaining_value': fields.float('Next Period Depreciation', digits_compute=dp.get_precision('Account'),required=True),
-        'depreciated_value': fields.float('Amount Already Depreciated', required=True),
-        'depreciation_date': fields.date('Depreciation Date', select=1),
-        'move_id': fields.many2one('account.move', 'Depreciation Entry'),
-        'move_check': fields.function(_get_move_check, method=True, type='boolean', string='Posted', store=True)
-    }
-
-    def create_move(self, cr, uid, ids, context=None):
-        context = dict(context or {})
-        can_close = False
-        asset_obj = self.pool.get('account.asset.asset')
-        period_obj = self.pool.get('account.period')
-        move_obj = self.pool.get('account.move')
-        move_line_obj = self.pool.get('account.move.line')
-        currency_obj = self.pool.get('res.currency')
+    name = fields.Char(string='Depreciation Name', required=True, index=True)
+    sequence = fields.Integer(required=True)
+    asset_id = fields.Many2one('account.asset.asset', string='Asset', required=True, ondelete='cascade')
+    parent_state = fields.Selection(related='asset_id.state', string='State of Asset')
+    amount = fields.Float(string='Current Depreciation', digits=dp.get_precision('Account'), required=True)
+    remaining_value = fields.Float(string='Next Period Depreciation', digits=dp.get_precision('Account'), required=True)
+    depreciated_value = fields.Float(string='Amount Already Depreciated', required=True)
+    depreciation_date = fields.Date('Depreciation Date', index=True)
+    move_id = fields.Many2one('account.move', string='Depreciation Entry')
+    move_check = fields.Boolean(compute='_get_move_check', string='Posted', track_visibility='always', store=True)
```

Comment:
are you sure that track_visibility='always' will not generate lots and lots of unuseful chatter messages?